### PR TITLE
update link texts

### PR DESCRIPTION
**Pull request by ctstacey**
"doctest/1" ->"doctest/2"
"test/2" ->"test/3"

### CODE REVIEW

File: getting-started/mix-otp/introduction-to-mix.markdown [hunk 1]
```diff
@@ -194,9 +194,9 @@ It is important to note a couple of things:
 
 2. we define a test module named `KVTest`, in which we [`use ExUnit.Case`](https://hexdocs.pm/ex_unit/ExUnit.Case.html) to inject the testing API;
 
-3. we use one of the injected macros, [`doctest/1`](https://hexdocs.pm/ex_unit/ExUnit.DocTest.html#doctest/2), to indicate that the `KV` module contains doctests (we will discuss those in a later chapter);
+3. we use one of the injected macros, [`doctest/2`](https://hexdocs.pm/ex_unit/ExUnit.DocTest.html#doctest/2), to indicate that the `KV` module contains doctests (we will discuss those in a later chapter);
```
Comment (josevalim) [inline getting-started/mix-otp/introduction-to-mix.markdown:197]:
```suggestion
3. we use one of the imported macros, [`doctest/2`](https://hexdocs.pm/ex_unit/ExUnit.DocTest.html#doctest/2), to indicate that the `KV` module contains doctests (we will discuss those in a later chapter);
```

File: getting-started/mix-otp/introduction-to-mix.markdown [hunk 2]
```diff
@@ -194,9 +194,9 @@ It is important to note a couple of things:
 
 2. we define a test module named `KVTest`, in which we [`use ExUnit.Case`](https://hexdocs.pm/ex_unit/ExUnit.Case.html) to inject the testing API;
 
-3. we use one of the injected macros, [`doctest/1`](https://hexdocs.pm/ex_unit/ExUnit.DocTest.html#doctest/2), to indicate that the `KV` module contains doctests (we will discuss those in a later chapter);
+3. we use one of the imported macros, [`doctest/2`](https://hexdocs.pm/ex_unit/ExUnit.DocTest.html#doctest/2), to indicate that the `KV` module contains doctests (we will discuss those in a later chapter);
```
Comment (josevalim) [inline getting-started/mix-otp/introduction-to-mix.markdown:197]:
```suggestion
3. we use one of the imported macros, [`doctest/1`](https://hexdocs.pm/ex_unit/ExUnit.DocTest.html#doctest/1), to indicate that the `KV` module contains doctests (we will discuss those in a later chapter);
```

File: getting-started/mix-otp/introduction-to-mix.markdown [hunk 3]
```diff
@@ -194,9 +194,9 @@ It is important to note a couple of things:
...
 
-4. we use the [`test/2`](https://hexdocs.pm/ex_unit/ExUnit.Case.html#test/3) macro to define a simple test;
+4. we use the [`test/3`](https://hexdocs.pm/ex_unit/ExUnit.Case.html#test/3) macro to define a simple test;
```
Comment (josevalim) [inline getting-started/mix-otp/introduction-to-mix.markdown:199]:
```suggestion
4. we use the [`test/2`](https://hexdocs.pm/ex_unit/ExUnit.Case.html#test/2) macro to define a simple test;
```

**Comment by josevalim** [comment]
After further consideration, I think the text is correct. We are using test/2 in the code, so I made the link mirror that (although it will point to the exact same place!).

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Comment by ctstacey** [comment]
Hi Jose,
Thanks for considering my pull request.
Perhaps I misunderstood the meaning of test/2.
Does this mean we are calling the function with 2 arguments, or the function expects two arguments, and how do default arguments effect the arity of the function?
Thanks in advance,
Charles.

**Comment by josevalim** [comment]
If you call test(foo, bar), then you are calling test/2, with two arguments. It happens that the test macro, with default arguments, is defining both test/2 and test/3. :)

**Comment by ctstacey** [comment]
Oh nice! Thank-you!
Am I right in assuming the definition of test/2 and test/3 happen in the compiler (at compile time)?
(I was looking but couldn't find anything that explains this in the documentation)

**Comment by josevalim** [comment]
Correct.

**Comment by ctstacey** [comment]
Thanks! Have a great day! 💚 💙 💜 💛 ❤️

**Pull request merged**